tsqa/test_cases.py

```diff
@@ -46,7 +46,7 @@ class EnvironmentCase(unittest.TestCase):
     def run(self, result=None):
         unittest.TestCase.run(self, result)
         # we want to keep track of failures at a class level-- not instance level
-        self.__class__.__successful &= result.result.wasSuccessful()
+        self.__class__.__successful &= result.wasSuccessful()
```

Comment:
Hmm, this might be a python 2/3 api change... Are you running python2 or 3? (the docs you linked is to python3), if thats what it is I can add a check and call the appropriate one